### PR TITLE
Updated docker-compose function

### PR DESCRIPTION
**Pull request by dixyushi**

### CODE REVIEW

File: documentation/Functions.md [hunk 1]
```diff
@@ -297,18 +297,32 @@ will run 'npm install' in the directory 'my-thai-star/angular'
 ***
 
 ### dockerCompose <a name="dockerCompose"></a>
+This function is used for docker compose.
```
Comment (GuentherJulian) [inline documentation/Functions.md:300]: This function is used to running [Docker Compose](https://docs.docker.com/compose/).

File: documentation/Functions.md [hunk 2]
```diff
@@ -297,18 +297,32 @@ will run 'npm install' in the directory 'my-thai-star/angular'
...
-startupTime = Time in seconds to wait before checking if the server is running
-port: Port on which the server is running
-path: The URL path on which is checked if the server is running
+Note:
+1. If the tutorial needs to be tested on the console environment, you have to specify a port.
 
-If the tutorial should be tested on the console environment, you have to specify a port.
+2. No background script will run. Katacoda user needs to execute it manually.
+
+3.  Docker should be installed.
```
Comment (GuentherJulian) [inline documentation/Functions.md:325]: Docker must be installed. In the Katacoda environment, Docker is already preinstalled.

**Review by GuentherJulian — CHANGES_REQUESTED** [review]

File: documentation/Functions.md [hunk 3]
```diff
@@ -297,18 +297,41 @@ will run 'npm install' in the directory 'my-thai-star/angular'
...
-startupTime = Time in seconds to wait before checking if the server is running
-port: Port on which the server is running
-path: The URL path on which is checked if the server is running
+Note:
+1. If the tutorial needs to be tested on the console environment, you have to specify a port.
 
-If the tutorial should be tested on the console environment, you have to specify a port.
+2. No background script will run. 
+
+3. The command for execution will be generated by katacoda runner, so user will have to execute this command manually.
```
Comment (GuentherJulian) [inline documentation/Functions.md:330]: Katacoda

File: documentation/Functions.md [hunk 4]
```diff
@@ -297,18 +297,41 @@ will run 'npm install' in the directory 'my-thai-star/angular'
...
+
+4. Docker must be installed. In the Katacoda environment, Docker is already preinstalled.
+
+5. TO use this function, devonfw-ide must be installed.
```
Comment (GuentherJulian) [inline documentation/Functions.md:334]: devonfw-ide has not to be installed. Only Docker which is preinstalled in Katacoda as you mentioned above.

**Review by GuentherJulian — CHANGES_REQUESTED** [review]

File: documentation/Functions.md (hunk 3 above)
Comment (dixyushi) [reply documentation/Functions.md:330]: done!

**Pull request merged**